static/app/views/issueDetails/groupDetails.tsx

```diff
@@ -741,6 +741,15 @@ function GroupDetailsPageContent(props: GroupDetailsProps & FetchGroupDetailsSta
     );
     return issueDetailsTourData?.seen ?? false;
   }, [assistantData]);
+  const isIssueDetailsTourAvailable = useMemo(() => {
+    if (!hasStreamlinedUI) {
+      return false;
+    }
+    return (
+      location.hash === '#tour' ||
+      organization.features.includes('issue-details-streamline-tour')
+    );
+  }, [hasStreamlinedUI, location.hash, organization.features]);
```

Comment:
doesn't necessarily need a useMemo since its a boolean

Reply:
chatted in Slack, I'll keep this one because of the features list check, but memo makes more sense now, thanks!